README.md

```diff
@@ -122,6 +122,28 @@ shipment_quote = Shipcloud::ShipmentQuote.create(
 shipment_quote.price # => 6.2
 ```
 
+### Create a pickup request
+
+To request parcels being picked up, you need to provide the carrier name and the time (earliest and latest) your shipments shall be fetched. You may also provide a list of shipment ids to specify only certain shipments to be included in the pickup request.
```

Comment:
OK consistence over naming. I think "id" is more precise than "name" but this might be an issue for another pr.

As a user of the API I should know the list of shipments may be empty.